src/calls.h

```diff
@@ -271,7 +271,7 @@ EXPORT_INLINE ObjFunc_1ARGS HDLR_XARGS(Obj func)
 **
 *F  IS_FUNC( <obj> )  . . . . . . . . . . . . . check if object is a function
```

Comment:
```suggestion
*F  IS_FUNC( <obj> )  . . . . . . . . . . . . . check if object <obj> is a function
```
Just a minor remark.

Reply:
Out of 3083 such comments ("short function description"), just 136 reference a variable in this way. 

I really think the comment is fine as it is (it conveys its meaning well, doesn't it?). In any case, if it is to be changed, I'd suggest to change it in more places, and in a PR dedicate to it.

Reply:
I guess it is also a personal preference. Yes, you're right, this I probably clear enough and we can leave it as is.